src/lightning/pytorch/profilers/simple.py

```diff
@@ -80,15 +80,24 @@ def stop(self, action_name: str) -> None:
     def _make_report_extended(self) -> Tuple[_TABLE_DATA_EXTENDED, float, float]:
         total_duration = time.monotonic() - self.start_time
         report = [
-            (a, np.mean(d), len(d), np.sum(d), 100.0 * np.sum(d) / total_duration)
+            (
+                a,
+                torch.mean(torch.tensor(d)).item(),
```

Comment:
Can you convert d to tensor just once? You'll need to convert this list comprehension into a loop